datafusion/functions-array/src/kernels.rs

```diff
@@ -893,6 +895,100 @@ pub fn array_length(args: &[ArrayRef]) -> Result<ArrayRef> {
     }
 }
 
+/// array_resize SQL function
```

Comment:
I think @jayzhan211  may have a plan to split these functions up into different modules (e.g. not one giant `kernels.rs` file). No changes needed for this PR, just FYI

Reply:
Thanks @alamb. Yes, our direction will be like that with the next PR.